README.md

```diff
@@ -38,3 +38,12 @@ Solve the problems and try again **gradle test**
 ```bash
 ./gradlew build
 ```
+# Test Coverage 
+```bash
+./gradlew build jacocoTestReport
+```
+Open the file **index.html**
+
+```bash
+build/reports/jacoco/test/html/index.html
```

Comment:
Hi @starsaminf , thanks for pushing this. Is this report supposed to be generated every time we run ./gradlew test?
When I try locally, the report is not generated unless I execute ./gradlew jacocoTestReport. It would be great if we can generate the report as part of ./gradlew test